src/lib/application-state-stores/explorer-stores.ts

```diff
@@ -0,0 +1,81 @@
+import { writable } from "svelte/store";
+import type { MetricsExplorerEntity } from "$lib/redux-store/explore/explore-slice";
+import type { MetricViewMetaResponse } from "$common/rill-developer-service/MetricViewActions";
+import type { TimeSeriesTimeRange } from "$common/database-service/DatabaseTimeSeriesActions";
+
+export interface MetricsExplorerStoreType {
+  entities: Record<string, MetricsExplorerEntity>;
+}
+export const MetricsExplorerStore = writable({
```

Comment:
I would try to break this out a bit. We want to remove the possibility that the consumer tries to set this store, which they currently can. How I tend to do this:

```
const { set, update, subscribe } = writable(...);
/** code logic etc. etc. *
export const MetricsExplorerStore = { subscribe };
```

All a svelte store needs to be coded as such is a `subscribe` method.

Reply:
Ya this feels better. Lemme update the PR

Reply:
Done